### PR TITLE
Added table for listing all the meetups.

### PR DESCRIPTION
**Pull request by sidntrivedi012**

### CODE REVIEW

**Comment by sidntrivedi012** [comment]
@darkHarry @V1shvesh PTAL.

**Comment by saranshbarua** [comment]
@sidntrivedi012 please add a screenshot.

**Comment by sidntrivedi012** [comment]
@saranshbarua 
screenshot from 2018-09-19 23-05-34

File: README.md [hunk 1]
```diff
@@ -6,7 +6,11 @@ Organizational repository for the meetups initiave in OSDC
...
-## Linux Meetup 0
+## Meetups
 
-Linux Introductory talk given by [crapsystem](https://github.com/crapsystem) (Himanshu Nailwal) in the first hour.
-A CTF based linux introduction game using simple shell commands was hosted in the final hour.
+List of all the meetups of the OSDC community.
+
+| S.No. | Date | Meetup Agenda | Details | 
+| :---: | :--: | ------ | ---------- |
+|   1   | 30-Aug-2018 | [Wanna Play Linux](./linux-meetup-0) | Linux Introductory talk given by [crapsystem](https://github.com/crapsystem) (Himanshu Nailwal) in the first hour.A CTF based linux introduction game using simple shell commands was hosted in the final hour.|
```
Comment (ryzokuken) [inline README.md]: @sidntrivedi012 Let's use `DD-MM-YY`?

File: README.md [hunk 2]
```diff
@@ -6,7 +6,11 @@ Organizational repository for the meetups initiave in OSDC
...
 * [Sudeepam97](https://github.com/Sudeepam97) - **P Sudeepam** &lt;`<sudeepam.pandey@gmail.com>`&gt;
 
-## Linux Meetup 0
+## Meetups
 
-Linux Introductory talk given by [crapsystem](https://github.com/crapsystem) (Himanshu Nailwal) in the first hour.
-A CTF based linux introduction game using simple shell commands was hosted in the final hour.
+List of all the meetups of the OSDC community.
+
+| S.No. | Date | Meetup Agenda | Details |
```
Comment (ryzokuken) [inline README.md]: Meetup title sounds better?

Comment (darkHarry) [reply README.md]: yes. Change this to Meetup Title.

File: README.md (hunk 1 above)
Comment (darkHarry) [reply README.md]: Current format looks good. Can you increase its width so that the whole date is in one line ?

**Comment by darkHarry** [comment]
There are plenty of ways to increase column width and i'm sure you will find one. Also the table in Readme doesn't look human readable in raw form, read up some Markdown style guide and improve it.
(https://github.com/carwin/markdown-styleguide)

**Comment by sidntrivedi012** [comment]
@darkHarry I am using &nbsp; for increasing the width of the column but it is making the table look haphazard.
 The main issue that the column is not widening is that the Details column has large content. So can we include the link to the blog of the specific meetup in the Fourth column i.e. by replacing Details?

**Pull request closed without merging**